dockerfiles/run/spring/Dockerfile

```diff
@@ -17,7 +17,7 @@ EXPOSE 25 110 143 465 587 993 4000
 
 WORKDIR /root
 
-ARG VERSION=3.5.0-SNAPSHOT
+ARG VERSION=3.6.0-SNAPSHOT
```

Comment:
Why do you have a release commit in this PR???

Reply:
That's required for fixing the build and it will disappear with a rebase on master